src/commands/create_command.py

```diff
@@ -30,3 +33,18 @@ def validate(schema_file, schema_dir):
     def check_database(self):
         if self.data[Keys.DATABASE] is None:
             raise WrongParameterError("No database detected")
+
+    @staticmethod
+    def create_table_schema(t_path, table):
```

Comment:
Something for you.
If you find yourself that you have a lot of functions to create something, think of create a class.

create table
Create Index

so it’s a time to think to have a table class